src/devtools/client/debugger/src/components/Editor/LineHitCounts.tsx

```diff
@@ -48,14 +51,15 @@ function LineHitCounts({ cm }: Props) {
   }, [dispatch, sourceId, hitCounts]);
 
   useLayoutEffect(() => {
-    if (!cm) {
+    if (!editor) {
       return;
     }
 
-    const doc = cm.editor.getDoc();
+    const doc = editor.editor.getDoc();
```

Comment:
Haha, I almost changed `cm` to `editor` too but then I felt weird writing `editor.editor` 😆

Reply:
Yeah, I figured that was the case! I couldn't help changing it seeing them side-by-side 😇